xml/cha_images.xml

```diff
@@ -470,55 +470,101 @@
+     addressed, all affected images are updated immediately and the three month
+     timer restarts with this forced replacement.
+    </para>
+    <para>
+     &suse; is committed to address all security vulnerabilities disclosed
+     through the <literal>Common Vulnerabilities and Exposures process</literal>
+     (CVE) and a score of 9.0 or greater in the <literal>Common Vulnerability
+      Scoring System</literal> (CVSS). For more information about the effects and
+     rating of CVEs, refer to the <link
+      xlink:href="https://www.suse.com/security/cve/">&suse; CVE database</link>.
+    </para>
+    <!-- alternative version
+    <para>
+     In the event of a severe security vulnerability disclosed through the 
+     <literal>Common Vulnerabilities and Exposures</literal> (CVE) process, the
+     &suse; public cloud development team will refresh all affected images to
+     ensure that new instances launch in a patched state.
+    </para>
+    <para>
+     In this context, severe security vulnerabilities are defined as CVEs with a
+     score of 9.0 or greater in the <literal>Common Vulnerability Scoring
+     System</literal> (CVSS). For more information about the effects and rating
+     of CVEs, refer to the <link
+     xlink:href="https://www.suse.com/security/cve/">&suse; CVE database</link>.
+    </para>
+    -->
+   </listitem>
+  </itemizedlist>
+
   <para>
-   Information about the effects of CVEs can always be found in the <link
-    xlink:href="https://www.suse.com/security/cve/">&suse; CVE database</link>.
+   The life cycle of an image consists of four different states:
   </para>
   <variablelist>
+   <title>&suse; public cloud image states</title>
    <varlistentry>
     <term>Active</term>
     <listitem>
      <para>
-      Active images are on a three month rolling refresh cycle.
+      Active images are fully supported and refreshed at least every three
+      months. The duration lasts until the image is replaced by a newer image
+      version.
      </para>
     </listitem>
    </varlistentry>
    <varlistentry>
     <term>Inactive</term>
     <listitem>
      <para>
-      Inactive images will only get refreshed for critical security updates.
+      Inactive images are supported unter the rules of LTSS and will only get
+      refreshed for critical security updates. The duration term is defined by
+      the product. For more information, refer to <link
+       xlink:href="https://www.suse.com/de-de/support/policy-products/#cloud"/>
      </para>
     </listitem>
    </varlistentry>
    <varlistentry>
     <term>Deprecated</term>
     <listitem>
      <para>
-      Deprecated images do not get refreshed. Images in the deprecated state
-      have entered the last six months of their life-cycle and are subject to
-      removal at the end of the six months period.
+      Deprecated images are no longer supported as they have entered the last
```

Comment:
This is where it gets complicated because the deletion policy is not consistent across providers. Here is my suggestion:
"""
Deprecated images may no longer be supported. The status of support depends on the support status of the product in the image. Deprecated images are not made available in regions added after an image has been set to deprecated and images do no longer get refreshed. At the end of the six month deprecation period images are subject to deletion. It is strongly discouraged to use images in the deprecated state for the creation of new instances.
"""

Reply:
Could we simplify this to
`Deprecated images are only supported if the product in the image is still supported.` (and then link to https://www.suse.com/lifecycle)